README.md

```diff
@@ -1,110 +1,39 @@
 [![Build Status](https://circleci.com/gh/mapbox/mapbox-gl-js.svg?style=svg)](https://circleci.com/gh/mapbox/mapbox-gl-js) [![Coverage Status](https://coveralls.io/repos/github/mapbox/mapbox-gl-js/badge.svg?branch=master)](https://coveralls.io/github/mapbox/mapbox-gl-js?branch=master)
```

Comment:
Should we remove or move the badges? They look odd hanging out at the top:

![image](https://cloud.githubusercontent.com/assets/98601/22078854/6e5c8b02-dd6e-11e6-97d6-fd4fa621d499.png)